Schema.js

```diff
@@ -75,6 +78,7 @@ function classNameIsValid(className) {
     className === '_Session' ||
     className === '_SCHEMA' || //TODO: remove this, as _SCHEMA is not a valid class name for storing Parse Objects.
     className === '_Role' ||
+    className === '_GlobalConfig' ||
```

Comment:
`ExportAdapter` is the legacy user of this class that needs to be fixed before we can remove `_SCHEMA` from the list of valid class names. Take a look at `schemas.js` for how do it without using the `classNameIsValid` function.